docs/request-custom.md

```diff
@@ -0,0 +1,210 @@
+# Implement a custom request
+
+This lesson describes how to implement your own custom request types, for types that
+don't have out-of-the-box Volley support.
+
+## Write a custom request
+
+Most requests have ready-to-use implementations in the toolbox; if your response is a string,
+image, or JSON, you probably won't need to implement a custom `Request`.
+
+For cases where you do need to implement a custom request, this is all you need
+to do:
+
+- Extend the `Request<T>` class, where `<T>` represents the type of parsed response
+  the request expects. So if your parsed response is a string, for example,
+  create your custom request by extending `Request<String>`. See the Volley
+  toolbox classes `StringRequest` and `ImageRequest` for examples of
```

Comment:
noting for myself that at some point, it'd be nice to link to actual Javadoc wherever we reference a class for the first time in a doc. We don't currently host it anywhere ourselves, but we could generate it and put it into the docs/ folder somewhere, or use a service like https://javadoc.io/doc/com.android.volley/volley